core/domain/blog_services.py

```diff
@@ -277,23 +258,30 @@ def get_blog_post_rights(blog_post_id, strict=True):
     return get_blog_post_rights_from_model(model)
 
 
-def get_blog_post_rights_by_user_id(user_id):
-    """Retrieves the rights object for all blog posts for which the given
-    user is an editor.
+def get_published_blog_post_summaries_by_user_id(user_id, max_limit):
+    """Retrieves the summary objects for given number of published blog posts
+    for which the given user is an editor.
 
     Args:
         user_id: str. ID of the user.
+        max_limit: int. The number of models to be fetched.
```

Comment:
Either the max_limit should be decided internally or this should be a keyword_param with max_limit=feconf.MAX...

Reply:
https://github.com/oppia/oppia/pull/13278#discussion_r664848923 This explains reason for this as well.